packages/profile-sync-controller/src/controllers/user-storage/network-syncing/types.ts

```diff
@@ -0,0 +1,6 @@
+import type { NetworkConfiguration } from '@metamask/network-controller';
+
+export type RemoteNetworkConfiguration = NetworkConfiguration & {
+  v: '1';
+  deleted?: boolean;
```

Comment:
Are you not going with reduced key length like we did for notifications & accounts? (for example `deleted` > `d`)
I was under the impression that you guys first did that for notifications in order to have a smaller payload

Reply:
Yep, as per the PRD we are actually going to perform a "soft delete", this is because it allows other devices (when performing the big sync) to know if we should delete local networks vs upload the remote networks

Reply:
Oooooh, yeah lets minimise the size.